src/encoded/static/libs/bootstrap/modal.js

```diff
@@ -233,15 +233,14 @@ export class ModalFooter extends React.Component {
         }
 
         return (
-            <div className="modal-footer">
+            <div className={`modal-footer${addCss ? ` ${addCss}` : ''}`}>
```

Comment:
If `addCss` is an empty string, then it’s false and the final string should just be "modal-footer." If it’s not an empty string because someone passed it in, then it has to be added with a preceding space. If we didn’t need the space it could just be a conditionless ${addCss}. We could also just add a space to "modal-footer ", but then when `addCss` is empty, then you’d see `class="modal-footer "` with an unneeded space after it.

Reply:
Ah okay, I see the space.